Volume/View Models/OrganizationContentViewModel.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  OrganizationContentViewModel.swift
+//  Volume
+//
+//  Created by Vin Bui on 11/17/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+extension OrganizationContentView {
+
+    @MainActor
+    class ViewModel: ObservableObject {
```

Comment:
More of a question for my own curiosity, is there a reason why you made this an extension of Org Content View rather than having a separate Org View Model class?

Reply:
It's basically saying, "This isn't just any view model. It's the view model for `OrganizationContentView`." There's also better encapsulation this way.